Formula/crackpkcs.rb

```diff
@@ -0,0 +1,30 @@
+class Crackpkcs < Formula
+  desc "Multithreaded program to crack PKCS#12 files"
+  homepage "https://crackpkcs12.sourceforge.io"
+  url "https://download.sourceforge.net/project/crackpkcs12/0.2.11/crackpkcs12-0.2.11.tar.gz"
+  sha256 "9cfd0aa1160545810404fff60234c7b6372ce7fcf9df392a7944366cae3fbf25"
+  license "GPL-3.0-or-later"
+
+  depends_on "pkg-config" => :build
+  depends_on "openssl@1.1"
+
+  resource "cert.p12" do
+    url "https://github.com/crackpkcs12/crackpkcs12/raw/9f7375fdc7358451add8b31aaf928ecd025d63d9/misc/utils/certs/usr0052-exportado_desde_firefox.p12"
```

Comment:
Which one is the official source? GitHub or Sourceforge?

Reply:
The tarballs are hosted on SourceForge (there are no tags or archives on GitHub). The repository is at GitHub.

I was told to write a test case for which I need a certificate. Since there is no way to access a certificate in the tarball in the test stage, I downloaded it. 

Then I was told not to download it via curl but to specify it as ressource. So I did that.

Reply:
Having the source in two locations is kinda weird but you can't fix that either. I made an upstream issue for it.